lib/bundle.js

```diff
@@ -74,11 +75,11 @@ module.exports = (targetBundle, cnfg, opts) => {
     })
   }
 
-  function bundle() {
+  function bundle(errorHandler) {
```

Comment:
Do you meant use an error handler that's passed in or the `watchErrorHandler` imported at the top of the file?